src/databricks/azext_databricks/azext_metadata.json

```diff
@@ -1,4 +1,4 @@
 {
-    "azext.isExprimental": true,
+    "azext.isExperimental": true,

```

Comment:
command test will not fail?

Reply:
This typo only affects the message shown when installing the extensions. Current tests will not catch it, but there is a plan to add metadata validation to avoid such issue.